xml/obs_best_practice_webui_usage.xml

```diff
@@ -155,10 +155,24 @@
    Now let's create a link to a package and add a repository to build
    against. A link is basically a pointer to sources of an already existing
    package in another project. By *repository*, we mean a build target -
```

Comment:
in another project -> at another location ?

There are also links pointing to the same project...